server.go

```diff
@@ -2,13 +2,14 @@ package scim
 
 import (
 	"fmt"
-	f "github.com/elimity-com/scim/internal/filter"
-	"github.com/scim2/filter-parser/v2"
 	"net/http"
 	"net/url"
 	"strconv"
 	"strings"
 
+	f "github.com/elimity-com/scim/internal/filter"
```

Comment:
This should be grouped with the other elimity-scim imports.   Is there precedent within the repo for importing this as "f"?